promql/parser/generated_parser.y.go

```diff
@@ -12,105 +12,121 @@ import (
+	strings     []string
+	series      []SequenceValue
+	histogram   *histogram.FloatHistogram
+	descriptors map[string]interface{}
+	bucket_set  []float64
+	int         int64
+	uint        uint64
+	float       float64
+	duration    time.Duration
 }
 
-const (
-	EQL                      = 57346
-	BLANK                    = 57347
-	COLON                    = 57348
-	COMMA                    = 57349
-	COMMENT                  = 57350
-	DURATION                 = 57351
-	EOF                      = 57352
-	ERROR                    = 57353
-	IDENTIFIER               = 57354
-	LEFT_BRACE               = 57355
-	LEFT_BRACKET             = 57356
-	LEFT_PAREN               = 57357
-	METRIC_IDENTIFIER        = 57358
-	NUMBER                   = 57359
-	RIGHT_BRACE              = 57360
-	RIGHT_BRACKET            = 57361
-	RIGHT_PAREN              = 57362
-	SEMICOLON                = 57363
-	SPACE                    = 57364
-	STRING                   = 57365
-	TIMES                    = 57366
-	operatorsStart           = 57367
-	ADD                      = 57368
-	DIV                      = 57369
-	EQLC                     = 57370
-	EQL_REGEX                = 57371
-	GTE                      = 57372
-	GTR                      = 57373
-	LAND                     = 57374
-	LOR                      = 57375
-	LSS                      = 57376
-	LTE                      = 57377
-	LUNLESS                  = 57378
-	MOD                      = 57379
-	MUL                      = 57380
-	NEQ                      = 57381
-	NEQ_REGEX                = 57382
-	POW                      = 57383
-	SUB                      = 57384
-	AT                       = 57385
-	ATAN2                    = 57386
-	operatorsEnd             = 57387
-	aggregatorsStart         = 57388
-	AVG                      = 57389
-	BOTTOMK                  = 57390
-	COUNT                    = 57391
-	COUNT_VALUES             = 57392
-	GROUP                    = 57393
-	MAX                      = 57394
-	MIN                      = 57395
-	QUANTILE                 = 57396
-	STDDEV                   = 57397
-	STDVAR                   = 57398
-	SUM                      = 57399
-	TOPK                     = 57400
-	aggregatorsEnd           = 57401
-	keywordsStart            = 57402
-	BOOL                     = 57403
-	BY                       = 57404
-	GROUP_LEFT               = 57405
-	GROUP_RIGHT              = 57406
-	IGNORING                 = 57407
-	OFFSET                   = 57408
-	ON                       = 57409
-	WITHOUT                  = 57410
-	keywordsEnd              = 57411
-	preprocessorStart        = 57412
-	START                    = 57413
-	END                      = 57414
-	preprocessorEnd          = 57415
-	startSymbolsStart        = 57416
-	START_METRIC             = 57417
-	START_SERIES_DESCRIPTION = 57418
-	START_EXPRESSION         = 57419
-	START_METRIC_SELECTOR    = 57420
-	startSymbolsEnd          = 57421
-)
+const EQL = 57346
+const BLANK = 57347
+const COLON = 57348
```

Comment:
I'd like to note for the record that this PR un-did the formatting applied to this generated file in #9563.

In my opinion this version is correct - we shouldn't be reformatting generated files - but I would have done the formatting change in its own PR for clarity in the history.

Reply:
I see. Yeah, generated files either stay unformatted, or the formatting has to be part of the auto-generation.
We should have caught that in #9563…